starterkits/basic/.storybook/preview-body.html

```diff
@@ -0,0 +1,5 @@
+<style>
+  body {
+    margin: 20px;
```

Comment:
could it be reason of misunderstanding?
where component starts? and its own padding?

Reply:
well, from my personal experience it is useful thing for storybook and don't break anything even with fixed position (for header) for example. This override created more visual readability of the component

Reply:
but what about pages template. i think it is better to add option to customize it per component?